internal/lang/eval_test.go

```diff
@@ -879,21 +879,9 @@ func Test_enhanceFunctionDiags(t *testing.T) {
 		},
 		{
 			"Broken prefix",
-			"attr = magic::foo::bar::extra::missing_function(54)",
+			"attr = magic::foo::bar::extra::extra2::missing_function(54)",
 			"Invalid function format",
-			"Expected provider::<provider_name>::<function_name>, instead found \"magic::foo::bar::extra::missing_function\"",
-		},
-		{
```

Comment:
how come we're deleting these tests?

Reply:
The functionality no longer exists within eval.go.  It will be hit much earlier on in context tests (which are currently commented out)